docs/topics/recipes.md

```diff
@@ -743,7 +743,7 @@ easy manner.
 ### Valid array keys for style `applyFromArray()`
 
 The following table lists the valid array keys for
-\PhpOffice\PhpSpreadsheet\Style applyFromArray() classes. If the "Maps
+Style applyFromArray() classes. If the "Maps
```

Comment:
btw, do you have a rough estimate on when you could finish this work ? I am thinking on releasing a beta "soonish"